src/ServerRequestFactory.php

```diff
@@ -497,7 +535,7 @@ private static function marshalProtocolVersion(array $server)
      * @param $cookieHeader
      * @return array
      */
-    private static function parseCookieHeader($cookieHeader)
+    public static function parseCookieHeader($cookieHeader)
```

Comment:
To be reverted - this is `private` API for which we shouldn't provide BC guarantees